recipes/pyjnius/activate.bat

```diff
@@ -0,0 +1,6 @@
+SET "PYJNIUS_JAR_BACKUP=%PYJNIUS_JAR%"
+SET "PYJNIUS_JAR=%CONDA_PREFIX%\share\pyjnius\pyjnius.jar"
+SET "JDK_HOME_BACKUP=%JDK_HOME%"
+SET "JDK_HOME=%JAVA_HOME%"
+SET "PATH_BACKUP=%PATH%"
+SET "PATH=%JDK_HOME%\jre\bin\server;%JDK_HOME%\bin;%PATH%"
```

Comment:
Since [`%JDK_HOME%` is just the `%LIBRARY_PREFIX%`]( https://github.com/conda-forge/openjdk-feedstock/blob/16d07e2287c7d295df58f6ad82a59c9d3885226e/recipe/scripts/activate.bat#L2 ), `%JDK_HOME%\bin` is already on the `%PATH%` as `conda` adds that for us. Given this, it appears the critical thing is having `%JDK_HOME%\jre\bin\server` added. Inspecting that directory reveals only one library, which is `jvm.dll` (needed by libraries using JNI). This explains the loading error and why we needed to add this to the `%PATH%`. Have submitted PR ( https://github.com/conda-forge/openjdk-feedstock/pull/30 ) to have `openjdk` do this for us.

Reply:
Another thing worth checking, it appears on [macOS and Linux set `JAVA_LD_LIBRARY_PATH` to the equivalent location]( https://github.com/conda-forge/openjdk-feedstock/blob/16d07e2287c7d295df58f6ad82a59c9d3885226e/recipe/scripts/activate.sh#L7-L11 ). Maybe setting this alone would also solve the issue on Windows?